app/src/main/java/com/mxt/anitrend/view/activity/index/MainActivity.kt

```diff
@@ -240,6 +243,7 @@ class MainActivity : ActivityBase<Void, BasePresenter>(), View.OnClickListener,
         mDrawerLayout.addDrawerListener(mDrawerToggle)
         mDrawerToggle.syncState()
         updateUI()
+        requestCurrentUser()
```

Comment:
`onSaveInstanceState` is usually called when this activity is about to get paused, suggesting a user is leaving the screen thus calling `requestCurrentUser` at this point would do the opposite of the **create/resume** behaviour you're after. Did you mean to have this run in `onResume`?

Reply:
This is onResume().
![image](https://user-images.githubusercontent.com/6830979/117552551-4f567000-b04c-11eb-8f22-6fb2825e6be2.png)